common.go

```diff
@@ -36,6 +36,14 @@ func GetLanguageExtensions(language string) []string {
 
 // GetLanguage return the Language for a given filename and file content.
 func GetLanguage(filename string, content []byte) string {
+	if lang, safe := GetLanguageByFilename(filename); safe {
+		return lang
+	}
+
+	if lang, safe := GetLanguageByShebang(content); safe {
```

Comment:
Yes it is, this it the reason because https://github.com/src-d/simple-linguist/pull/22 must be merged before this PR. This PR is branched from the other PR, from commit f45efec to back the commits are the same that in https://github.com/src-d/simple-linguist/pull/22